client/platform/desktop/backend/serializers/viame.ts

```diff
@@ -254,31 +255,44 @@ async function parse(input: Readable, imageMap?: Map<string, number>): Promise<A
       if (err !== undefined) {
         reject(err);
       }
-      resolve({ tracks: Array.from(dataMap.values()), fps });
+      const tracks = Array.from(dataMap.values());
+
+      if (imageMap !== undefined) {
+        if (missingImages.length > 0 && missingImages.length !== tracks.length) {
```

Comment:
Same as other comment.  There is no relationship between the number of tracks and the missingImages, unless there is a single detection per frame.  This would fail for image-sequences composed of multiframe tracks that don't have filenames.  Like on the python version I'd suggest we modify the tests to account for that as well.